modules/ROOT/pages/custom-resolvers.adoc

```diff
@@ -45,7 +45,7 @@ The inclusion of the fields `firstName` and `lastName` in the `requires` argumen
 """Informs @neo4j/graphql that a field will be resolved by a custom resolver, and allows specification of any field dependencies."""
 directive @customResolver(
     """Fields that the custom resolver will depend on."""
-    requires: [String!]
+    requires: String
```

Comment:
Surely this is a mandatory argument? Or no?

Reply:
I don't think it's mandatory because you can require no fields. These lines in  also seem to suggest we don't require it: https://github.com/neo4j/graphql/blob/18bff81ccf7ab5a7b52b3e1fd7f6bbe930eeb22b/packages/graphql/src/schema/get-custom-resolver-meta.ts#L74C1-L80C6

Reply:
Also this is the actual directive definition: https://github.com/neo4j/graphql/blob/18bff81ccf7ab5a7b52b3e1fd7f6bbe930eeb22b/packages/graphql/src/graphql/directives/customResolver.ts#L23